amundsen_application/static/js/components/Feedback/FeedbackForm/RatingFeedbackForm/index.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+import AbstractFeedbackForm, { DispatchFromProps, StateFromProps } from '../../FeedbackForm';
+
+import { GlobalState } from "../../../../ducks/rootReducer";
+import { submitFeedback, resetFeedback } from '../../../../ducks/feedback/reducer';
+
+export class RatingFeedbackForm extends AbstractFeedbackForm {
+  constructor(props) {
+    super(props)
+  }
+
+  renderCustom() {
+    const ratings = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+    const radioButtonSet = ratings.map(rating => (
```

Comment:
this functionality could be placed in a separate function like `renderRadioButtonSet`, and be called directly `{ this.renderRadioButtonSet() }`. Would be really handy to write the unit tests.

Reply:
It looks like `renderCustom()` was already split out for this purpose.